### PR TITLE
tabs: Document the next tab design

### PR DESCRIPTION
**Pull request by JBWatenbergScality**
**Component**:

Tabs

**Description**:

We are reviewing core-ui component and studying API and implementations of those next components

### CODE REVIEW

File: stories/tabs.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <Tab path={string} label={} textBadge={} >{children}</Tab>
+    <Tab path={string} label={} textBadge={} >{children}</Tab>
+    <Tab path={string} label={} textBadge={} >{children}</Tab>
+</Tabs>
+```
+
+Tabs are based on react router and extends the current route architecture of the application with child routes.
+Each path is taken as a child path to the current router path.
+
+Tab size depend on the label, minimum size is 5rem.
```
Comment (Cuervino) [inline stories/tabs.stories.mdx:24]: Tab **height** is fixed, it's the tab **width** that is dependent on the label string length.

File: stories/tabs.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Next Version Implementation Guides/Components/Navigation/Tabs" />
+
+
+## Implementation guidelines for next tabs component 
+
+It should use text basic  component to display the labels
```
Comment (Cuervino) [inline stories/tabs.stories.mdx:8]: TextBasic
color TextPrimary for Active tab
color TextSecondary for Non-active tabs

File: stories/tabs.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Next Version Implementation Guides/Components/Navigation/Tabs" />
+
+
+## Implementation guidelines for next tabs component 
+
+It should use text basic  component to display the labels 
+The latest tabs should display the separation line at the end to mark the end of the tab
```
Comment (Cuervino) [inline stories/tabs.stories.mdx:9]: There should be also a separation line between each tabs

**Comment by ChengYanJin** [comment]
@Cuervino can you please send us the figma design link so we can attach it to the design doc?

**Pull request closed without merging**